custom_code/alertstream_handlers.py

```diff
@@ -301,9 +300,26 @@ def handle_einstein_probe_alert(message, metadata):
             skymap['PROBDENSITY'].unit = '1 / sr'
             calculate_credible_region(skymap, localization)
             calculate_footprint_probabilities(skymap, localization)
+    
+    ep_ra = alert.get('ra')
+    ep_dec = alert.get('dec')
+    ep_name = alert['id'][0]
+    ep_error = alert.get('ra_dec_error')
+    t_ep = Target.objects.create(name=ep_name, type='SIDEREAL', ra = ep_ra, dec = ep_dec)
```

Comment:
After making this, maybe you should create an `EventCandidate` to link it to the `NonLocalizedEvent` right away.